src/universal/redux/reducers/currentTemplate/actionCreators.js

```diff
@@ -29,3 +29,14 @@ export function promoteTemplateToProductionVersion(projectId, templateId, versio
     }
   };
 }
+
+export function loadTemplateVersionHistory(versionId) {
```

Comment:
I don't get this name. It seems to load an older version, not a list of historical versions. I think what is does is actually load(Older)Version, although it can load the current one can't it?